Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -601,75 +601,94 @@ internal string LogoPathFromUri(string uri)
                 // windows 8.1 https://msdn.microsoft.com/en-us/library/windows/apps/hh965372.aspx#target_size
                 // windows 8 https://msdn.microsoft.com/en-us/library/windows/apps/br211475.aspx
 
-                string path;
-                if (uri.Contains("\\"))
-                {
-                    path = Path.Combine(Package.Location, uri);
-                }
-                else
-                {
-                    // for C:\Windows\MiracastView etc
```

Comment:
Not sure why we needed it. C:\Windows\MiracastView shouldn't be a uwp program right?

Reply:
Maybe it is in some older version of win10

Reply:
I can't even find it on my pc.

Reply:
I neither

Reply:
Check whether file exists? That shall be good.

Reply:
Check the new commit please.